README.md

```diff
@@ -1,279 +1,43 @@
-
-Index adapters have to implement the following methods for `ember-deploy` to be able to use them:
-
-####`upload([bootstrapIndexHTML])`
-
-  __Parameters__
-
-  _bootstrapIndexHTML_
-
-  `upload` will get passed the content of `dist/index.html` that gets build from ember-cli. Feel free to do whatever you need to do with its file content.
-
-  This method has to return a `RSVP.Promise`.
-
-####`activate([revisionToActivate])`
-
-__Parameters__
-
-_revisionToActivate_
-
-`activate` takes a revision you pass to it and activates it in the key-value store. The `redis`-adapter for example takes the passed revision-key and sets it as the value of the `<project.name>:current`-key in redis. Of course you can do whatever you want in this method. If you decide that you don't need a notion of activating revisions it is recommended that you nevertheless return a useful message to the user in this method.
-
-This method has to return a `RSVP.Promise`.
-
-####`list`
-
-`list` lists all uploaded revisions you have deployed and that you want the user to be able to preview or to rollback to. Of course you can do whatever you want in this method. If you decide that you don't need a notion of listing revisions it is recommended that you nevertheless return a useful message to the user in this method.
-
-This method has to return a `RSVP.Promise`.
-
-### Asset-Adapters
-
-`asset-adapters` take care of publishing your project assets. In the default use case (Luke Melia's lightning approach) your assets will be published to a static asset server like AWS S3. If you want to use some other static asset server service or you want to do something entirely different you can write your own `asset-adapter`. Asset-Adapters have to implement the following method for ember-deploy to know what to do:
-
-####`upload`
-
-Ember deploy will gzip your `js` and `css` assets and copy these gzipped assets and all other assets from ember-cli's `dist`-folder to the `tmp/assets-sync`-directory. You can do whatever you want in an `asset-adapter`'s `upload`-method but most likely you will upload these assets to some kind of static-asset server with the `upload`-method.
-
-This method has to return a `RSVP.Promise`.
-
-### Tagging-Adapters
+## Commands
 
-`tagging-adapters` take care of tagging revisions when uploading revisions via the `deploy-index`-command. `ember-deploy` comes bundled with the `sha`-tagging-adapter that will tag your revisions according to the current git-sha.
+ember-cli-deploy adds the following deployment commands to Ember CLI:
 
-Tagging Adapters need to implement the following method for ember-deploy to know what to do:
+- `ember deploy` Initiates the deployment pipeline, invoking any registered ember-cli-deploy plugins at the defined hook points
```

Comment:
Have only added `ember deploy` at this stage as I'm unclear as to what is actually needed other than that. How will ember activate actually fit in if it's is really only relevant to one potential deployment strategy (the lightening method).